### ISSUE TITLE
First release in 2021?

### ISSUE THREAD

**Issue by okravets** [microsoft/vcpkg#17028]
When first release planned in 2021?
Thank you,
Olexandr

**Comment by NancyLi1013** [comment]
Hi @vicroms 

Are you clear something about the plan to release?

Thanks.

**Comment by vicroms** [comment]
We have separated releases for `vcpkg-tool` (the vcpkg executable) and the vcpkg catalog. 

I assume this is about the latter. I want to point out that we recommend users to use the latest sources from the `vcpkg` repository instead of depending on these "release". 

The generated release tags are arbitrary, usually we did one every month but sometimes we forgot to do them, and since there isn't any special meaning behind when we decide to do them and no automated process to generate them, depending on new "releases" is unreliable.

If a user needs a specific revision of vcpkg, our recommendation is to checkout a specific commit or consider enabling versioning  and using the [`builtin-baseline` field](https://github.com/microsoft/vcpkg/blob/master/docs/examples/versioning.getting-started.md).

Finally, I'll take a look at generating a new release tag this week.
